dev.env

```diff
@@ -0,0 +1,6 @@
+SPHINX_PORT=9312
+SPHINX_EFS=/var/local/efs-dev/geodata/service-sphinxsearch/dev/index/
+PGPASS=www-data
+PGUSER=www-data
```

Comment:
Although those credential are not really real one (easily guessable) as a best practice we should never put credential in a git repository in plain text. Might be a good idea to use `gopass` here ? Not sure though if it will make the workflow for sphinxsearch too complicated if using gopass ?

Reply:
These env variables are part of the infra definition and should live in the vhost setup for service search in `infra-vhost` repository